### PR TITLE
LPS-143659

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java [hunk 1]
```diff
@@ -252,15 +252,14 @@ public User addDefaultAdminUser(
 
 		long creatorUserId = 0;
 		boolean autoPassword = false;
+		boolean resetPassword = _isPasswordReset(companyId);
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java:255]: Seems like the var name here should be `passwordReset`.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Looks great @pei-jung @alvarosaugarlr. Just the one var naming comment, otherwise this looks good to go

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
ci:test:relevant

**Comment by epark18** [comment]
Looks like it could be a CI issue, but a lot of tests are not running. Going to re-run relevant to check if failures are consistent.

**Comment by epark18** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
@epark18 are the test failures related?

**Comment by epark18** [comment]
@pei-jung There are some issues with some builds not running on CI, so I can't say for sure. I will follow up with CI about this and rerun again.

**Comment by pei-jung** [comment]
thanks @epark18!

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**